lib/widgets/menu/header.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+import 'package:movie_card/shared/ui_consts.dart';
+
+class Header extends StatelessWidget {
+  const Header({
+    super.key,
+  });
+
+  static const String title = "TMDB";
+  static const String logo = 'assets/logo/logoTMDB.png';
+  static const double padding = 25.0;
+  static const double height = 30;
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Padding(
+        padding: const EdgeInsets.symmetric(
+          horizontal: padding,
+        ),
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            Row(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Text(
+                  title,
+                  style: titleStyle,
+                ),
+                Image.asset(
+                  logo,
+                  height: height,
+                ),
+              ],
+            ),
+            IconButton(
+              onPressed: () {
+                showSearch(
+                  context: context,
+                  delegate: SearchMovieDelegate(),
+                );
+              },
+              icon: Icon(
+                Icons.search_rounded,
+                color: colors.background,
+                size: height,
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class SearchMovieDelegate extends SearchDelegate {
```

Comment:
Nice addition! I'm not seeing this in action though 😔.
Either include that on the video, or (I see it's a work in progress) leave a `// TODO` comment or so, so we know it's not ready yet.

Reply:
I was expecting something more... Expressive, you know 😅 
Like `// TODO - Implement movie search`
That `// TODO` seems related to the `title`, not the whole feature. Nevermind